server/testsupport/createInductionDtoTestDataBuilder.ts

```diff
@@ -35,13 +36,13 @@ const aLongQuestionSetCreateInductionDto = (
     previousWorkExperiences: {
       hasWorkedBefore:
         !options || options.hasWorkedBefore === null || options.hasWorkedBefore === undefined
-          ? true
+          ? HasWorkedBeforeValue.YES
```

Comment:
All of these changes to the test data builders (6 files in total I think) can be simplified if you want.

What you've done works, but was originally written in this fairly long hand way because the previous field was a `boolean` and because of javascript's horrible falsey-ness! 😬 

The idea with these test data builder functions is that they take an optional `options` object, which in turn contains some optional properties. If the `options` object, or a given property is not set, then we use a sensible default.

Now that your property is no longer a boolean, it means you can do this:
```
    hasWorkedBefore: options?.hasWorkedBefore || HasWorkedBeforeValue.YES
    ....
    experiences: options?.hasWorkedBefore === HasWorkedBefore.Yes
        ? .....
        : ....
```

Much simpler 😁 

I think you can make this same change to all 6 test data builder files you've changed here - entirely up to you if you want to do it (because what you have does still work 👍 )